rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -995,6 +995,31 @@ public void onSuccess(String result) {
         }
     }
 
+    public void undeployNodeSource() {
+        if (model.getSelectedNodeSource() != null) {
+            String nodeSourceName = model.getSelectedNodeSource().getSourceName();
+
+            rm.undeployNodeSource(LoginModel.getInstance().getSessionId(),
+                                  model.getSelectedNodeSource().getSourceName(),
+                                  new AsyncCallback<String>() {
+                                      @Override
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)